Makefile

```diff
@@ -1,25 +1,42 @@
 LOG_LEVEL ?= spin=trace
 CERT_NAME ?= local
 SPIN_DOC_NAME ?= new-doc.md
+export PATH := target/debug:target/release:$(HOME)/.cargo/bin:$(PATH)
 
 ARCH = $(shell uname -p)
 
 ## dependencies for e2e-tests
-E2E_VOLUME_MOUNT     ?=
-E2E_BUILD_SPIN       ?= false
-E2E_TESTS_DOCKERFILE ?= e2e-tests.Dockerfile
-MYSQL_IMAGE          ?= mysql:8.0.22
-REDIS_IMAGE          ?= redis:7.0.8-alpine3.17
-POSTGRES_IMAGE       ?= postgres:14.7-alpine
-REGISTRY_IMAGE       ?= registry:2
+E2E_SPIN_RELEASE_VOLUME_MOUNT   ?=
+E2E_SPIN_DEBUG_VOLUME_MOUNT     ?=
```

Comment:
What's the reason for keeping this separate? Wouldn't a better interface be for the user to specify which spin binary they want to run, and then we copy that binary into the docker container instead of mounting the entire directory?

Reply:
but what would be the fallback if spin binary path is not provided by user

Reply:
We could still prob for a binary using the same order that you currently do (debug, release, .cargo/bin, $PATH) but instead of mounting the various locations of the various spin binaries directly into the volume, we could instead copy the binary to a new location and mount just that directory that just contains the spin binary.

This is just an idea to make the set up a bit easier to follow since we have several mounting options.

Reply:
Ah you're right, sorry. I think in general, I'm really bad at Make, and I'd like to simplify this wherever possible.

Reply:
hi @rylev, do u have an alternate approach in mind or the current change looks ok to you?